source/_posts/2022-02-02-release-20222.markdown

```diff
@@ -0,0 +1,6908 @@
+
+More information can be found in
+[Architectural Decission Record 0019](https://github.com/home-assistant/architecture/blob/master/adr/0019-GPIO.md).
+
+Additionally, there are many projects around that handle these purposes of GPIO
+better than Home Assistant. For example, our sister project ESPHome, or
+the MQTT IO project, and many more.
+
+We do not forbid the use of GPIO, but we are merely deprecating and removing
+built-in integrations, providing GPIO functionality from Core. We welcome
+custom integrations (existing or new ones) to provide alternatives. However,
+for most cases we recommend on using dedicated hardware, like an ESP device,
+instead.
+
+The following integrations have been deprecated and will be removed in
+Home Assistant Core **2022.4**:
+
+- BeagleBone Black GPIO
+- BH1750
+- Blinkt!
+- Bosch BME280 Environmental Sensor
+- Bosch BME680 Environmental Sensor
+- Bosch BMP280 Environmental Sensor
+- DHT Sensor
+- Enviro pHAT
+- HTU21D(F) Sensor
+- mcp23017
+- MH-Z19 CO2 Sensor
+- Orange Pi GPIO
+- pcal9535a
+- pi4ioe5v9xxxx
+- PiFace Digital I/O (PFIO)
+- Piglow
+- pigpio Daemon PWM LED
+- Raspberry Pi RF
+- Rasphihats
+- Sense HAT
+- Sensirion SHT31
+- Time of Flight
+
+The following integration have been deprecated and will be removed in
+Home Assistant Core **2022.6**:
+
+- 1-Wire (SysBus only!)
+- Raspberry Pi GPIO
+
+Note: 1-Wire is a special case, as it supports both the SysBus and connecting
+to a remote owfs server. The use of the SysBus is deprecated, however,
+connecting to a remote owfs server is not.
+
+{% enddetails %}
+
+{% details "Binary Sensors" %}
+
+Previously, `binary_sensor` entities could have the state `on`, `off`, and
+in case the device was unreachable: `unavailable`.
+
+However, compared to other entities, it wasn't able to have the `unknown` state,
+which has been added this release.
+
+So, a `binary_sensor` entity can now have the `on`, `off`, `unavailable`, or `unknown` state.
+
+You might need to adapt your automations or scripts to take this new
+`unknown` state into account.
+
+([@frenck] - [#60193]) ([binary_sensor docs])
+
+{% enddetails %}
+
+{% details "Groups: Locks" %}
+
+The behavior of grouped locks has been reversed such that "unlocked" is
+mapped to `on` and "locked" to off.
+
+This means that:
+
+- A group with mixed domains will be considered `on` if any lock included in
+  the group is unlocked.
+- A group of locks will be considered "unlocked" if any lock included in the
+- group is unlocked.
+
+([@emontnemery] - [#62580]) ([lock docs]) ([group docs])
+
+{% enddetails %}
+
+{% details "Fan/Humidifer/Light/Remote/Siren/Switch/Vacuum" %}
+
+Previously, toggle-based entities (like `fan`, `light`, `switch`, `remote`, `siren`, `vacuum`, `humidifier`) could have the state `on` or `off`, and in case the device was unreachable: `unavailable`. However, compared to other entities, toggle entities aren't able to have an `unknown` state, this now has changed.
```

Comment:
```suggestion
Previously, toggle-based entities (like `fan`, `light`, `switch`, `remote`, `siren`, `vacuum`, `humidifier`) could have the state `on` or `off`, and in case the device was unreachable: `unavailable`. However, compared to other entities, toggle entities weren't able to have an `unknown` state, this now has changed.
```